### ISSUE TITLE
Element for IRL/MSL asset/sticker number?

### ISSUE THREAD

**Issue by rebeccahawke** [MSLNZ/equipment-register-schema#16]
In our equipment register we include the IRL/CI/MSL asset/sticker number. Where would this information belong in the schema?

**Comment by jborbely** [comment]
Perhaps add it to `<documentation>` as an optional element?

The relevant parts from the MSL Quality Manual states:
> 4.3.6 Equipment registers
   ...
   Other information may be included, as appropriate, such as:
   • Company asset register number

Currently, `<documentation>` is the element that we are adding the _other_ items listed in clause `4.3.6` to.

`<documentation>` may still need to be renamed. I personally don't like the name, but I don't know what to rename it to.

**Comment by jborbely** [comment]
Fixed in b7dead6e483826b7809fa8c35b25ec68fe19b9ac

It is now a child element to `<documentation>`

```xml
<documentation>
  <assetNumber>123456789</assetNumber>
</documentation>
```

b7dead6e483826b7809fa8c35b25ec68fe19b9ac also made all child elements to `<documentation>` be optional and they may be defined in any order.
